coordinator/src/message.rs

```diff
@@ -48,10 +49,21 @@ pub fn spawn_delivering_messages_to_authenticated_users(
         let traders = authenticated_users.clone();
         async move {
             let mut user_feed = tx_user_feed.subscribe();
-            while let Ok(new_user_msg) = user_feed.recv().await {
-                traders
-                    .write()
-                    .insert(new_user_msg.new_user, new_user_msg.sender);
+            loop {
+                match user_feed.recv().await {
+                    Ok(new_user_msg) => {
+                        traders
+                            .write()
+                            .insert(new_user_msg.new_user, new_user_msg.sender);
+                    }
+                    Err(RecvError::Closed) => {
+                        tracing::error!("New user message sender died! Channel closed.");
+                        break;
+                    }
```

Comment:
Eventually we could try to re-subscribe to the channel here.

Reply:
Even if the sender died? I think resubscribing would only make sense if the sender is also able to restart.

Reply:
True. Makes sense.